agent/hcp/telemetry/filter.go

```diff
@@ -0,0 +1,51 @@
+package telemetry
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+
+	"github.com/hashicorp/go-multierror"
+)
+
+// filterList holds a map of filters, i.e. regular expressions.
```

Comment:
I see a single item and not a map? Is this an old comment, or are we missing an implementation here?